HPXMLtoOpenStudio/resources/defaults.rb

```diff
@@ -5591,6 +5821,317 @@ def self.get_ceiling_fan_months(weather)
+        next if !system_ids.include?(ventilation_fan.id)
+        next if ventilation_fan.is_shared_system
+
+        if [HPXML::LocationKitchen, HPXML::LocationBath].include?(ventilation_fan.fan_location)
+          watts += ventilation_fan.count * ventilation_fan.fan_power
+        elsif not ventilation_fan.fan_power.nil?
+          watts += ventilation_fan.fan_power
+        else
+          watts += 3000 # FIXME: base-mechvent-cfis-no-additional-runtime.xml, e.g., has no FanPower defaulted
+        end
+      end
+      breaker_spaces += 1 # intentionally outside the ventilation_fans loop
+    elsif type == HPXML::ElectricPanelLoadTypePermanentSpaHeater
+      hpxml_bldg.permanent_spas.each do |permanent_spa|
+        next if !system_ids.include?(permanent_spa.heater_id)
+        next if ![HPXML::HeaterTypeElectricResistance, HPXML::HeaterTypeHeatPump].include?(permanent_spa.heater_type)
+
+        watts += 1000 # FIXME
+        breaker_spaces += 2
+      end
+    elsif type == HPXML::ElectricPanelLoadTypePermanentSpaPump
+      hpxml_bldg.permanent_spas.each do |permanent_spa|
+        next if !system_ids.include?(permanent_spa.pump_id)
+
+        watts += 1491
+        breaker_spaces += 1
+      end
+    elsif type == HPXML::ElectricPanelLoadTypePoolHeater
+      hpxml_bldg.pools.each do |pool|
+        next if !system_ids.include?(pool.heater_id)
+        next if ![HPXML::HeaterTypeElectricResistance, HPXML::HeaterTypeHeatPump].include?(pool.heater_type) # FIXME: probably want to separate out HeatPump here
+
+        watts += 27000
+        breaker_spaces += 2
+      end
+    elsif type == HPXML::ElectricPanelLoadTypePoolPump
+      hpxml_bldg.pools.each do |pool|
+        next if !system_ids.include?(pool.pump_id)
+
+        watts += 1491
+        breaker_spaces += 1
+      end
+    elsif type == HPXML::ElectricPanelLoadTypeWellPump
+      hpxml_bldg.plug_loads.each do |plug_load|
+        next if plug_load.plug_load_type != HPXML::PlugLoadTypeWellPump
+        next if !system_ids.include?(plug_load.id)
+
+        if hpxml_bldg.building_construction.number_of_bedrooms <= 3
+          watts += 746
+        else
+          watts += 1119
+        end
+        breaker_spaces += 2
+      end
+    elsif type == HPXML::ElectricPanelLoadTypeElectricVehicleCharging
+      hpxml_bldg.plug_loads.each do |plug_load|
+        next if plug_load.plug_load_type != HPXML::PlugLoadTypeElectricVehicleCharging
+        next if !system_ids.include?(plug_load.id)
+
+        # FIXME: next if MF?
+
+        if voltage == 120 # Level 1
+          watts += 1650
+        else # Level 2
+          watts += 7680
+          breaker_spaces += 1
+        end
+        breaker_spaces += 1
+      end
+    elsif type == HPXML::ElectricPanelLoadTypeLighting
+      watts += 3 * hpxml_bldg.building_construction.conditioned_floor_area
+      breaker_spaces += 0
+    elsif type == HPXML::ElectricPanelLoadTypeKitchen
+      watts += 3000
+      breaker_spaces += 0
```

Comment:
shouldn't breaker spaces be 2 for the two kitchen 1.5 kVA small appliance circuits?

Reply:
For both lighting and kitchen, I didn't see these as nonzero in the work plan document.

Reply:
@joseph-robertson - I interpreted each of the two kitchen branches as needing a breaker space (looking at the line in Table 1 of the Workplan). 

@lixiliu - can you confirm?